modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,97 @@
 
 [id="understanding-upgrade-channels_{context}"]
 
-= Upgrade channels and release paths
-Cluster administrators can configure the upgrade channel from the web console.
-
+= Update channels
 
 [id="candidate-version-channel_{context}"]
```

Comment:
you'll want to move this anchor to follow your shifting of the `candidate-4.y` section.